tests/phpunit/api/v4/Action/BaseCustomValueTest.php

```diff
@@ -34,15 +35,16 @@ abstract class BaseCustomValueTest extends UnitTestCase {
    */
   public function setUp(): void {
     $this->setUpOptionCleanup();
-    $cleanup_params = [
-      'tablesToTruncate' => [
-        'civicrm_custom_group',
-        'civicrm_custom_field',
-      ],
-    ];
-
-    $this->dropByPrefix('civicrm_value_my');
-    $this->cleanup($cleanup_params);
```

Comment:
Further simplification: this entire `setUp` function can be removed, and the lines `{ setUp as setUpOptionCleanup; }` can go as well.

Reply:
I did wonder about that - should it go in teardown or just go?

Reply:
@colemanw sinc this has passed how about we merge & you can put the answer to ^^ in a follow up PR?

Reply:
Removing it in this class it would still get called by the parent setUp function.